app/models/manageiq/providers/nuage/inventory/parser/network_manager.rb

```diff
@@ -65,6 +66,16 @@ def security_groups
     end
   end
 
+  def floating_ips
+    collector.floating_ips.each do |ip|
+      persister.floating_ips.find_or_build(ip['ID']).assign_attributes(
+        :address        => ip['address'],
+        :network_router => persister.network_routers.lazy_find(ip['parentID']),
```

Comment:
Can you explain this bit? Why both `network_router` and `cloud_tenant` use the same data from the floating IP?

Reply:
Oh, I missed the fact that you query the network router for its `cloud_tenant` in the next line. Thanks!